Bottom: ac89d9e57cb56c28aafce43703810b692da2775a
Top:    fc04b994e87daa4398be0943c6a4cd1bed35a71d
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-17 17:50:10 +0800

Refresh of fixup

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 776bad6..b8e5c2d 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -818,6 +818,7 @@ static int add_recvbuf_small(struct virtnet_info *vi, struct receive_queue *rq,
 
 	buf = (char *)page_address(alloc_frag->page) + alloc_frag->offset;
 	get_page(alloc_frag->page);
+	alloc_frag->offset += len;
 	sg_init_one(rq->sg, buf + NET_IP_ALIGN + xdp_headroom,
 		    vi->hdr_len + GOOD_PACKET_LEN);
 	err = virtqueue_add_inbuf(rq->vq, rq->sg, 1, buf, gfp);
